src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -209,9 +209,9 @@ public abstract partial class SymmetricAlgorithm : System.IDisposable
     {
         protected int BlockSizeValue;
         protected int FeedbackSizeValue;
-        protected byte[] IVValue;
+        protected byte[]? IVValue;
         protected int KeySizeValue;
-        protected byte[] KeyValue;
+        protected byte[]? KeyValue;
         protected System.Security.Cryptography.KeySizes[] LegalBlockSizesValue;
         protected System.Security.Cryptography.KeySizes[] LegalKeySizesValue;
```

Comment:
These don't need to be nullable?

Reply:
It can, seems intent is non null and derived classes should set this value with non null instance, I thought if make it nullable devs might think they can set it with null in derived class. Do you think it better to be nullable?

Reply:
How about marking it as [MaybeNull] instead then?